packages/azure-kusto-data/README.md

```diff
@@ -43,6 +43,22 @@ const kcsb = KustoConnectionStringBuilder.withAadApplicationCertificateAuthentic
     "certificate",
     "authorityId"
 );
+
+Option 2.2: Authenticating using AAD application id and corresponding certificate with SNI public key.
+Concat the private and publiccertificates
+```javascript
+const kcsb = KustoConnectionStringBuilder.withAadApplicationCertificateAuthentication(
+    `https://${clusterName}.kusto.windows.net`,
+    "appid",
+    "-----BEGIN CERTIFICATE-----
```

Comment:
Doesn't it accept a file path(instead of the string inline)?

Reply:
yes it does, and simply loads it ..  but this better shows how the content of the file should be

Reply:
How can the user understand it from this?

Specify that you need to create a file with that content and then give th epath

Reply:
you dont have to

Reply:
Are you sure? where is that documented? 
![image](https://github.com/Azure/azure-kusto-node/assets/6424271/c73fd22a-861f-4db8-ba15-164cc80ce8b1)

Azure identity says path to file system

Reply:
first- this works - but you can see here - there are two opetions .. 
![image](https://github.com/Azure/azure-kusto-node/assets/32278684/a6f20885-64e5-4365-9ae5-e286588f745f)

Reply:
ok